test/stdlib/AnyHashableCasts.swift.gyb

```diff
@@ -118,6 +118,16 @@ AnyHashableCasts.test("${valueExpr} as ${coercedType} as? ${castType}") {
 }
 % end
 
+AnyHashableCasts.test("Casting to AnyHashable doesn't leak") {
+  do {
+    let tracked = LifetimeTracked(42)
+    let anyHashable = AnyHashable(tracked)
+    let anyObject = anyHashable as AnyObject
```

Comment:
This is testing two different cases. Aren't there 3 types of cast consumption? I think you will need a full on address only type to test that, no? Maybe my memory is wrong.